src/Adapter/Product/QueryHandler/GetProductSuppliersForEditingHandler.php

```diff
@@ -49,6 +49,10 @@ public function handle(GetProductSuppliersForEditing $query): array
 
         /** @var ProductSupplierEntity $productSupplier */
         foreach (ProductSupplierEntity::getSupplierCollection($productIdValue, false) as $productSupplier) {
+            if ($query->combinationsOnly() && !$productSupplier->id_product_attribute) {
```

Comment:
Not sure I understand the purpose here..

Reply:
you can get suppliers for standard product or for combinations. 
If you get it for combinations by default you will still see one additional productSupplier which is associated with product id without combination :sweat: 
idk why its done this way, but Im afraid to dig deeper, so I just have to filter that one out here.